src/display/svg.js

```diff
@@ -782,9 +785,16 @@ SVGGraphics = (function SVGGraphicsClosure() {
         current.tspan.setAttributeNS(null, 'fill', current.fillColor);
       }
 
+      // Include the text rise in the matrix since the `pm` function creates
+      // the SVG element's `translate` entry. Work on a copy to not alter the
+      // original text matrix.
+      let matrix = current.textMatrix.slice();
+      if (current.textRise !== 0) {
+        matrix[5] += current.textRise;
```

Comment:
How often is this called? If `setTextRise` is called quite often, then you could relieve the GC by only making a copy when it is needed:

```javascript
let matrix = current.textMatrix;
if (current.textRise) {
    matrix = matrix.slice();
    matrix[5] += current.textRise;
}
```